### DISCUSSION TITLE
Monitors don't save Resolution after relogin

### DISCUSSION THREAD

**Discussion by BesterNameYT** [i3/i3#5959] [Q&A]
Hey everyone,

I Recently switched to Linux and Started trying out I3 on my Notebook without problems. After using it on my PC though I ran into a Annoying Problem. When using I3, both of my Monitor's Resolutions keep resetting to 800p. 
I have made a xrandr command which sets up both my Monitors Correctly, besides the fact that my 2nd Monitor stays at 24hz, which I still don't know why.
`xrandr --output HDMI-A-1 --primary --mode 1920x1200 --rate 59.95 --output HDMI-A-0 --mode 1920x1080 --rate 59.95 --above HDMI-A-1`
I have tried adding the command to my /etc/X11/xinit/xinitrc file with no result. I also looked around in various other Threads from Different Forums where I either get told to Edit a Config file that I don't have or where the Change doesn't help.
I'm really at my wits end with this, I have been looking for a Solution the Entire Afternoon with no Result.

This is my First time Using Linux outside of my Steam Deck so I'm not Familiar with any configs/settings, the Above mentioned Command already took me 30 Minutes to get working.

Any help would be gladly appreciated.

**Comment by orestisfl** [comment]
I would take a look at https://wiki.archlinux.org/title/multihead#Configuration_using_xorg.conf for xorg configuration and https://wiki.archlinux.org/title/Xrandr#Adding_undetected_resolutions for your refresh rate issue.

[autorandr](https://github.com/phillipberndt/autorandr) is another alternative for automating monitor setups.

This might be an issue with drivers. What kind of GPU do you use and which driver package?

**Reply by BesterNameYT** [reply]
I tried autorandr and It sadly didn't work. Neither did the Adding undetected resolution
`
leander@leander:~$ xrandr --addmode HDMI-A-1 1920x1200_60.00
leander@leander:~$ xrandr --output HDMI-A-1 1920x1200_60.00
xrandr: unrecognized option '1920x1200_60.00'
Try 'xrandr --help' for more information.
`
As to the xorg.conf stuff. I don't have that file, the only file in that Folder is a Keyboard config.
And I don't know if it matters but I'm using KDE Neon, since the Tutorials were on the Arch Forum.

**Reply by orestisfl** [reply]
> xrandr --output HDMI-A-1 1920x1200_60.00

You forgot `--mode`

> As to the xorg.conf stuff. I don't have that file

Yes, you create the files from scratch.

>  And I don't know if it matters but I'm using KDE Neon, since the Tutorials were on the Arch Forum.

Xorg is around for a while so usually it doesn't matter

**Reply by BesterNameYT** [reply]
my monitor config should now look like this
```
Section "Monitor"
  Identifier "HDMI-A-1"
  Modeline "1920x1200_60.00"  193.25  1920 2056 2256 2592  1200 1203 1209 1245 -hsync +vsync
  Option "PreferredMode" "1920x1200_60.00"
  Option "Below" "DP"
  Option "Primary" "true"
  Option "DPMS" "true"
EndSection

Section "Monitor"
  Identifier "HDMI-A-0"
  Modeline "1920x1080_60.00"  173.00  1920 2048 2248 2576  1080 1083 1088 1120 -hsync +vsync
  Option "PreferredMode" "1920x1080_60.00"
  Option "Above" "DVI"
  Option "DPMS" "true"
EndSection
```

Problem is, it won't save. I save the file via Kate but when I reopen it it's gone.
I also used X11 to get the Monitor names this time since they had a Different name in Wayland, dunno if that will help.
EDIT: Switching to X11 also got the whole xrandr stuff to work, so I think it helped.

**Reply by BesterNameYT** [reply]
Okay so my Top monitor now works fine (HDMI-A-0) but my Main and Bottom Monitor (HDMI-A-1) is pitch black.
